src/components/SiteSearch.vue

```diff
@@ -124,14 +122,17 @@ export default class SiteSearch extends Vue {
   quickLinks!: any[];
 
   get suggestedLinks() {
-    return this.quickLinks.filter(i => {
-      let findResult: any = this.searchData.find(
-        data => data.name === i.item.name
+    return this.quickLinks.reduce((acc, i) => {
+      let jsonSearchIndex = this.jsonSearch.findIndex(
+        (data) => data.name === i.item.name
       );
-      let suggestResult: any = this.searchData.indexOf(findResult);
-      this.quickLinkLoc.push(suggestResult);
-      return suggestResult;
```

Comment:
The real problem here is that `suggestedResult` can be `-1`, which is where the error was coming from